README.md

```diff
@@ -100,7 +100,7 @@ You can pass the following options via cli arguments, every options has the corr
 | Socket to listen on | `-s` | `--socket` | `FASTIFY_SOCKET` |
 | Log level (default to fatal) | `-l` | `--log-level` | `FASTIFY_LOG_LEVEL` |
 | Prints pretty logs | `-P` | `--pretty-logs` | `FASTIFY_PRETTY_LOGS` |
-| Watch process.cwd() directory for changes, recursively; when that happens, the process will auto reload. | `-w` | `--watch` | `FASTIFY_WATCH` |
+| Watch process.cwd() directory for changes, recursively; when that happens, the process will auto reload. more detail for [fastify-cli.config.js](#fastify-cliconfigjs)| `-w` | `--watch` | `FASTIFY_WATCH` |
```

Comment:
This should read `See [fastify-cli.config.js](#fastify-cliconfigjs) for more details`